onnxruntime/core/providers/openvino/ov_versions/data_ops.cc

```diff
@@ -0,0 +1,879 @@
+    {VERSION_2020_4,{"All"},"LeakyRelu"},
+    {VERSION_2020_4,{"All"},"Less"},
+    {VERSION_2020_4,{"All"},"Log"},
+    {VERSION_2021_2,{"MYRIAD"},"Loop"},
+    {VERSION_2020_4,{"All"},"LRN"},
+    {VERSION_2020_4,{"All"},"LSTM"},
+    {VERSION_2020_4,{"All"},"MatMul"},
+    {VERSION_2020_4,{"All"},"Max"},
+    {VERSION_2020_4,{"All"},"MaxPool"},
+    {VERSION_2020_4,{"All"},"Mean"},
+    {VERSION_2020_4,{"All"},"Min"},
+    {VERSION_2020_4,{"All"},"Mul"},
+    {VERSION_2020_4,{"All"},"Neg"},
+    {VERSION_2021_1,{"All"},"NonMaxSuppression"},
+    {VERSION_2021_1,{"CPU", "MYRIAD"},"NonZero"},
+    {VERSION_2021_1,{"All"},"Not"},
+    {VERSION_2020_4,{"CPU", "GPU"},"Not"},
+    {VERSION_2020_4,{"All"},"OneHot"},
+    {VERSION_2020_4,{"All"},"Pad"},
+    {VERSION_2020_4,{"All"},"Pow"},
+    {VERSION_2020_4,{"All"},"PRelu"},
+    {VERSION_2021_2,{"MYRIAD"},"Range"},
+    {VERSION_2020_4,{"All"},"Reciprocal"},
+    {VERSION_2020_4,{"CPU", "MYRIAD"},"ReduceLogSum"},
+    {VERSION_2020_4,{"All"},"ReduceMax"},
+    {VERSION_2020_4,{"All"},"ReduceMean"},
+    {VERSION_2020_4,{"All"},"ReduceMin"},
+    {VERSION_2020_4,{"CPU"},"ReduceProd"},
+    {VERSION_2020_4,{"All"},"ReduceSum"},
+    {VERSION_2020_4,{"CPU", "MYRIAD"},"ReduceSumSquare"},
+    {VERSION_2020_4,{"All"},"Relu"},
+    {VERSION_2020_4,{"CPU"},"Resize"},
+    {VERSION_2020_4,{"All"},"Reshape"},
+    {VERSION_2021_1,{"All"},"RoiAlign"},
+    {VERSION_2021_2,{"MYRIAD"},"Round"},
+    {VERSION_2021_1,{"MYRIAD"},"Scatter"},
+    {VERSION_2021_2,{"MYRIAD"},"ScatterElements"},
+    {VERSION_2020_4,{"CPU", "GPU"},"Selu"},
+    {VERSION_2020_4,{"All"},"Shape"},
+    {VERSION_2020_4,{"All"},"Sigmoid"},
+    {VERSION_2020_4,{"CPU"},"Sign"},
+    {VERSION_2020_4,{"CPU"},"Sign"},
+    {VERSION_2020_4,{"CPU"},"Sinh"},
+    {VERSION_2020_4,{"MYRIAD"},"SinFloat"},
+    {VERSION_2020_4,{"All"},"Slice"},
+    {VERSION_2020_4,{"All"},"Softmax"},
+    {VERSION_2020_4,{"All"},"SpaceToDepth"},
+    {VERSION_2020_4,{"All"},"Split"},
+    {VERSION_2020_4,{"All"},"Sqrt"},
+    {VERSION_2020_4,{"All"},"Squeeze"},
+    {VERSION_2020_4,{"CPU"},"Softsign"},
+    {VERSION_2020_4,{"All"},"Sub"},
+    {VERSION_2020_4,{"All"},"Sum"},
+    {VERSION_2020_4,{"CPU", "GPU"},"Tan"},
+    {VERSION_2020_4,{"All"},"Tanh"},
+    {VERSION_2021_2,{"MYRIAD"},"Tile"},
+    {VERSION_2020_4,{"All"},"Transpose"},
+    {VERSION_2020_4,{"All"},"TopK"},
+    {VERSION_2020_4,{"All"},"Unsqueeze"},
+    {VERSION_2021_1,{"CPU", "GPU"},"Upsample"},
+    {VERSION_2021_2,{"MYRIAD"},"Where"},
+    
+};
+
+void Capability::populate_types_supported() {
+
+    std::set<int> supported_types_initializer = {
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_BOOL,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_FLOAT,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_INT32,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_INT64,
+    };
+    
+    std::set<int> supported_types_vpu = {
```

Comment:
we need to add **ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_FLOAT16 for** VPU and GPU

Reply:
changes done

Reply:
reolved